### PR TITLE
ffmpeg: bump to 6.1.1

### PR DESCRIPTION
**Pull request by kenmays**
Tested on Haiku R1B4 x64.

### CODE REVIEW

File: media-video/ffmpeg/ffmpeg6-6.1.1.recipe [hunk 1]
```diff
@@ -4,12 +4,12 @@ and stream audio and video. It includes libavcodec - the leading audio/video \
...
-	2003-2023 the FFmpeg developers"
+	2003-2024 the FFmpeg developers"
 LICENSE="GNU LGPL v3"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
-CHECKSUM_SHA256="488c76e57dd9b3bee901f71d5c95eaf1db4a5a31fe46a28654e837144207c270"
-PATCHES="ffmpeg-$portVersion.patchset"
+CHECKSUM_SHA256="8684f4b00f94b85461884c3719382f1261f0d9eb3d59640a1f4ac0873616f968"
+PATCHES="ffmpeg-6.1.patchset"
```
Comment (jmairboeck) [inline media-video/ffmpeg/ffmpeg6-6.1.1.recipe:12]: I think the patchset file should be renamed instead of changing the name here.

Comment (kenmays) [reply media-video/ffmpeg/ffmpeg6-6.1.1.recipe:12]: We usually do it that way, but it is an unnecessary commit/change if nothing changes there (and it is within the same major/minor revision).

Comment (jmairboeck) [reply media-video/ffmpeg/ffmpeg6-6.1.1.recipe:12]: Git detects that the file was renamed and keeps the history, so there is no problem with renaming the file. The recipe file is also renamed without any problem.

Comment (korli) [reply media-video/ffmpeg/ffmpeg6-6.1.1.recipe]: Please name it ffmpeg-6.1.1.patchset

**Comment by Begasus** [comment]
Needed to run the tests, in `TEST()`:
```
	export LIBRARY_PATH="$sourceDir/libavutil${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libavcodec${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libswresample${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libavdevice${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libavfilter${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libavformat${LIBRARY_PATH:+:$LIBRARY_PATH}"
	export LIBRARY_PATH="$sourceDir/libswscale${LIBRARY_PATH:+:$LIBRARY_PATH}"
```

File: media-video/ffmpeg/ffmpeg6-6.1.1.recipe (hunk 1 above)
Comment (kenmays) [reply media-video/ffmpeg/ffmpeg6-6.1.1.recipe]: Fixed.

**Pull request merged**